BME68X gas sensor driver using BSEC (#11103)
nrfconnect/sdk-nrf: doc/nrf/drivers/bme68x_iaq.rst

.. _bme68x_iaq:

BME68X IAQ driver
#################

.. contents::
   :local:
   :depth: 2

You can use the BME68X IAQ driver to run the Bosch Sensor Environmental Cluster (BSEC) library in order to get Indoor Air Quality (IAQ) readings.

.. note::
   The BSEC library is distributed with a Bosch proprietary license (`BSEC license`_) that blocks it from been distributed as part of our nRF Connect SDK.

> To start using it you have to accept license and enable the download with the following commands:

Reviewer: ```suggestion
To start using it you have to accept the license and enable the download with the following commands:
```

> .. note::
>    The BSEC library is distributed with a Bosch proprietary license (`BSEC license`_) that blocks it from been distributed as part of our nRF Connect SDK.
> 
> To start using it you have to accept the license and enable the download with the following commands:
> 
> .. code-block::
> 
>    west config manifest.group-filter +bsec
>    west update

Reviewer 2: The note could be removed in this case because line no: 15 is a continuation of line no:13.
```suggestion
The BSEC library is distributed with a Bosch proprietary license (`BSEC license`_) that prevents it from being part of |NCS|.
To start using it, you have to accept the license and enable the download with the following commands:

.. code-block::

   west config manifest.group-filter +bsec
   west update
```

.. _bme68x_iaq:

BME68X IAQ driver
#################

.. contents::
   :local:
   :depth: 2

You can use the BME68X IAQ driver to run the Bosch Sensor Environmental Cluster (BSEC) library in order to get Indoor Air Quality (IAQ) readings.

.. note::

>    Using the BSEC library requires accepting a separate license agreement.

Reviewer 3: Want to be even more precise on that this is a different license and needs to be handled/ downloaded by customer.
Suggested text: 
The BSEC library is distributed with a Bosch proprietary [license](https://www.bosch-sensortec.com/media/boschsensortec/downloads/software/bme688_development_software/2023_04/license_terms_bme688_bme680_bsec.pdf). that blocks it from been distributed as part of our nRF Connect SDK. To start using it you have to accept license and enable the download as described.
